noah/src/main/scala/edu/uci/ics/cloudberry/noah/feed/TwitterFeedStreamDriver.java

```diff
@@ -0,0 +1,138 @@
+package edu.uci.ics.cloudberry.noah.feed;
+
+import com.google.common.collect.Lists;
+import com.twitter.hbc.ClientBuilder;
+import com.twitter.hbc.core.Client;
+import com.twitter.hbc.core.Constants;
+import com.twitter.hbc.core.endpoint.Location;
+import com.twitter.hbc.core.endpoint.StatusesFilterEndpoint;
+import com.twitter.hbc.core.processor.StringDelimitedProcessor;
+import com.twitter.hbc.httpclient.auth.Authentication;
+import com.twitter.hbc.httpclient.auth.OAuth1;
+import org.kohsuke.args4j.CmdLineException;
+import org.kohsuke.args4j.CmdLineParser;
+
+import java.io.*;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.concurrent.BlockingQueue;
+import java.util.concurrent.LinkedBlockingQueue;
+
+public class TwitterFeedStreamDriver {
+
+    static Client twitterClient;
+    volatile static boolean isConnected = false;
+
+    static StreamFeedSocketAdapterClient socketAdapterClient;
+
+    public static void run(Config config)
+            throws InterruptedException, IOException {
+        BlockingQueue<String> queue = new LinkedBlockingQueue<String>(10000);
```

Comment:
Sorry for didn't give you any hints on the legacy code. Could you check the twitter lib to verify if it is a good choice? Thanks!